HISTORY.md

```diff
@@ -1,3 +1,8 @@
+# 5.2.2 / 2023-01-18
```

Comment:
Just noticed there are some 'old' commits like this one in this PR. Does a `git rebase 6.x` get rid of them?